package.json

```diff
@@ -23,7 +23,7 @@
   "author": "Brandon Keepers",
   "license": "ISC",
   "dependencies": {
-    "@octokit/rest": "15.1.7",
+    "@octokit/rest": "15.1.9",
```

Comment:
Anyone opposed to switching this to `^15.0.0` so we don't have to bump this every time a new version is released?

cc @gr2m

Reply:
yes because we use internal APIs right now, I’m sorry for the noise, once we are on v16 the plugin/hook APIs will be public, but they might change until then